hypha/core/management/commands/initialize.py

```diff
@@ -0,0 +1,46 @@
+import click
```

Comment:
This need to be added to requirements-dev.txt

Reply:
Done